Packs/MicrosoftGraphIdentityandAccess/Integrations/MicrosoftGraphIdentityandAccess/MicrosoftGraphIdentityandAccess.py

```diff
@@ -255,6 +330,162 @@ def remove_member_from_role(client: Client, args: dict) -> str:
     return f"User ID {user_id} has been removed from role {role_object_id}"
 
 
+def ip_named_location_get(ms_client: Client, args: dict) -> CommandResults:
+    if ip_id := args.get('ip_id'):
+        if results := ms_client.get_ip_named_location(ip_id):
+            context = {
+                'id': ip_id,
+                'display_name': results.get('displayName'),
+                'time_created': results.get('createdDateTime'),
+                'time_modified': results.get('modifiedDateTime'),
+                'is_trusted': results.get('isTrusted'),
+                'ip_ranges': results.get('ipRanges')
+            }
+            return CommandResults(
+                'MSGraph.conditionalAccess.namedIpLocations',
+                'namedIpLocations',
+                outputs=context,
+                raw_response=results,
+                readable_output=tableToMarkdown(
+                    f'Ip named location \'{ip_id}\':',
+                    context
+                )
+            )
+    return CommandResults(readable_output=f"No ip location found for {ip_id}")
+
+
+def ip_named_location_update(ms_client: Client, args: dict) -> CommandResults:
+    if ip_id := args.get('ip_id'):
+        if results := ms_client.get_ip_named_location(ip_id):
+            data = {
+                '@odata.type': '#microsoft.graph.ipNamedLocation',
+                'displayName': results.get('displayName'),
+                'isTrusted': results.get('isTrusted'),
+                'ipRanges': results.get('ipRanges')
+            }
+            ips = args.get('ips')
+            is_trusted = args.get('is_trusted')
+            display_name = args.get('display_name')
+            if ips is not None:
+                ips = ms_ip_string_to_list(ips)
+                data['ipRanges'] = ips
+            if is_trusted is not None:
+                data['isTrusted'] = is_trusted
+            if display_name is not None:
+                data['displayName'] = display_name
+            ms_client.update_ip_named_location(ip_id, data)
+            return CommandResults(
+                'MSGraph.conditionalAccess.namedIpLocations',
+                'namedIpLocations',
+                outputs={},
+                raw_response={},
+                readable_output=tableToMarkdown(
+                    f'Trying to update ip named location \'{ip_id}\':', {'Success': 'True'})
+            )
+    return CommandResults(readable_output=tableToMarkdown(f'Trying to update ip named location \'{ip_id}\':',
+                                                          {'Success': 'True'}))
+
+
+def ip_named_location_create(ms_client: Client, args: dict) -> CommandResults:
+    ips = args.get('ips')
+    is_trusted = args.get('is_trusted')
+    display_name = args.get('display_name')
+    if ips is not None and display_name is not None and is_trusted is not None:
```

Comment:
If they are marked as required, This is redundant.
```suggestion
    if ips and display_name and is_trusted:
```